lib/puppetlabs_spec_helper/rake_tasks.rb

```diff
@@ -220,7 +220,7 @@
 PuppetSyntax.exclude_paths << 'spec/fixtures/**/*'
 PuppetSyntax.exclude_paths << 'pkg/**/*'
 PuppetSyntax.exclude_paths << 'vendor/**/*'
```

Comment:
Whilst you're at it, could you also add `.vendor/**/*`?  This is already in the puppet-lint ignore_paths, but missing here.

Reply:
Hi @alexjfisher, I see your point, why would we have Puppet Code in a `.vendor` repo, every time I have used a `.vendor` it has been for Ruby Gems. I am fine making this change but for Puppet Syntax I am scratching my head as to why we would hide modules in `.vendor`